diff --git a/node_modules/vfile/lib/index.js b/node_modules/vfile/lib/index.js
index 68db7f9..e29e940 100644
--- a/node_modules/vfile/lib/index.js
+++ b/node_modules/vfile/lib/index.js
@@ -9,9 +9,9 @@
  */
 
 import {VFileMessage} from 'vfile-message'
-import {minpath} from '#minpath'
-import {minproc} from '#minproc'
-import {urlToPath, isUrl} from '#minurl'
+import {minpath} from './minpath.browser.js'
+import {minproc} from './minproc.browser.js'
+import {urlToPath, isUrl} from './minurl.browser.js'
 
 /**
  * Order of setting (least specific to most), we need this because otherwise
